Makefile

```diff
@@ -9,7 +9,7 @@
 #   make deb-src -------------- produce a DEB source
 #   make deb ------------------ produce a DEB
 #   make docs ----------------- rebuild the manpages (results are checked in)
-#   make tests ---------------- run the tests (see test/README.md for requirements)
+#   make tests ---------------- run the tests (see https://docs.ansible.com/ansible/dev_guide/testing_integration_legacy.html for requirements)
```

Comment:
Running `make tests` runs unit tests via `ansible-test`, not the legacy integration tests, so this should link to: https://docs.ansible.com/ansible/dev_guide/testing_units.html